src/main/java/edu/cornell/kfs/module/purap/document/validation/impl/CuCreditMemoDocumentPreRules.java

```diff
@@ -97,37 +97,37 @@ protected boolean checkWireTransferTabState(final VendorCreditMemoDocument cmDoc
         return tabStatesOK;
     }
 
-    protected boolean hasWireTransferValues(final CreditMemoWireTransfer cmWireTransfer) {
+    protected boolean hasWireTransferValues(final PaymentSourceWireTransfer wireTransfer) {

```

Comment:
Another method that could be replaced with `PaymentSourcePreRulesService` usage.

Reply:
Can you please check whether this method is still being used at all? I think `PaymentSourcePreRulesServiceImpl` is invoking its own variant of it, rather than invoking the one on the document PreRules instance.